package/src/completionCacheManager/completionCacheManager.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { test, describe, expect, beforeEach, afterEach } from '@jest/globals';
+import { CompletionCacheManager, GetFromLanguageService, createCompletionCacheManager } from './completionCacheManager';
+import * as monaco from 'monaco-editor';
+import * as ls from 'vscode-languageserver-types';
+
+describe('completionCacheManager', () => {
+    let completionCacheManager: CompletionCacheManager;
+    const mockGetFromLanguageService: jest.MockedFunction<GetFromLanguageService> = jest.fn();
+    const resource = {} as monaco.Uri;
+    const position = {} as ls.Position;
```

Comment:
Nit: Do these need to be fake? Could we pass things that type-check to make the test more durable?

Reply:
Fakes are test doubles that implement business rules. In contrast, dummies are objects that are passed around but never actually used. They typically fill parameter lists when no operations are needed on that object. This scenario is ideal for using dummies. Dummies help simplify the test setup by acting as placeholders for required objects that are not the focus of the test. In this case, the focus of the test is whether the languageService was called or not.